src/content/structured/get-started/migrating-to-v3.mdx

```diff
@@ -116,6 +120,10 @@ Given a directory, it will scan over files and find any relevant ICDS components
 - `ic-multi-select` (`IcMultiSelect`) is no longer a standalone component. The multi-select functionality can now be achieved using the `ic-select` (`IcSelect`) component and its `multiple` prop.
```

Comment:
I've noticed that this should be "ic-select-with-multi" - do you mind changing this as well?
```suggestion
- `ic-select-with-multi` (`IcSelectWithMulti`) is no longer a standalone component. The multi-select functionality can now be achieved using the `ic-select` (`IcSelect`) component and its `multiple` prop.
```